src/service/action-impl.js

```diff
@@ -748,9 +748,10 @@ function isActionWhitelisted_(invocation, whitelist) {
   const {method, node, tagOrTarget} = invocation;
   const tagOrTargetCalled = tagOrTarget.toLowerCase();
   let methodCalled = method.toLowerCase();
-  const defaultActionAlias = node.getDefaultActionAlias();
+  const defaultActionAlias = (typeof node.getDefaultActionAlias == 'function')
```

Comment:
I think this can be cleaner:

```js
const {method, node, tagOrTarget} = invocation;
// Use alias if default action was invoked.
if (method === DEFAULT_ACTION && typeof node.getDefaultActionAlias === 'function') {
  method = node.getDefaultActionAlias();
}
const lcMethod = method.toLowerCase();
const lcTagOrTarget = tagOrTarget.toLowerCase();
return whitelist.some(w => {
  return (lcMethod === w.method.toLowerCase() 
      && lcTagOrTarget === w.tagOrTarget.toLowerCase());
});
```

Reply:
done. Thanks.